src/odin_sequencer/manager.py

```diff
@@ -44,6 +45,8 @@ def __init__(self, path_or_paths=None):
         self.provides = {}
         self.context = {}
         self.file_paths = {}
+        self._auto_reload = False
```

Comment:
We should be consistent about use of leading underscore in internal member variables. Python doesn't enforce access controls on this, it's just a hint to developers. It is however useful on methods, since wildcard `import *` in code using a module (or __init__.py to control importing an entire package) doesn't import underscore prefixed methods. So I'd vote for not prefixing members, but prefixing methods only used internally.

Reply:
Sure! I am guessing the same applies for the classes inside the `watcher.py` module?